DEVGUIDE.md

```diff
@@ -173,6 +180,160 @@ It's important to use `Release` if you want to see if your changes have had a no
 
 Use the `Debug` configuration to test your changes locally. It is the default. Do not use the `Release` configuration! Local development and testing of Visual Studio tooling is not designed for the `Release` configuration.
 
+#### Writing and running benchmarks
```

Comment:
Maybe it's better to change the levels? Headlines are hard to read

![image](https://user-images.githubusercontent.com/2669927/165090608-1a259216-528c-4947-b771-5a603aae3fce.png)

Reply:
I've changed structure a bit, bumped some headings, should be easier to read now.